products/nextjs.md

```diff
@@ -11,6 +11,11 @@ versionCommand: npx next --version
 changelogTemplate: https://github.com/vercel/next.js/releases/tag/v__LATEST__
 releaseDateColumn: true
 
+identifiers:
+-   purl: pkg:npm/next.js
+-   purl: pkg:github/vercel/next.js
+-   purl: pkg:github/zeit/next.js
```

Comment:
https://github.com/zeit/next.js now redirect to https://github.com/vercel/next.js, should we really keep it ?